tasks/Module 4/task_7.py

```diff
@@ -0,0 +1,21 @@
+"""Excel-to-CSV Converter."""
+import os
+import csv
+import openpyxl
+
+
+def xl_to_csv(xl_file):
+    """Excel-to-CSV Converter."""
```

Comment:
It is general comment regarding docstrings. This docstring is almost the same as function name and thus doesn't give additional information from point of view of person who is reading the code. It would be much better to have more descriptive docstrings. Usually the best way is to start docstring with a verb:
`"""Convert Excel worksheet to CSV files by writing each sheet into separate CSV file."""`.